tox.ini

```diff
@@ -38,13 +38,13 @@ deps =
     ruff==0.0.206
 commands =
     codespell {toxinidir}/. \
-    --skip {toxinidir}/.git \
-    --skip {toxinidir}/.tox \
-    --skip {toxinidir}/build \
-    --skip {toxinidir}/lib \
-    --skip {toxinidir}/venv \
-    --skip {toxinidir}/.mypy_cache \
-    --skip {toxinidir}/icon.svg
+        --skip {toxinidir}/.git \
+        --skip {toxinidir}/.tox \
+        --skip {toxinidir}/build \
+        --skip {toxinidir}/lib \
+        --skip {toxinidir}/venv \
+        --skip {toxinidir}/.mypy_cache \
+        --skip {toxinidir}/icon.svg
```

Comment:
You can also have codespell the same as me in my `tox.ini` file:

```
codespell {toxinidir}/. --skip {toxinidir}/.git --skip {toxinidir}/.tox \
      --skip {toxinidir}/build --skip {toxinidir}/lib --skip {toxinidir}/venv \
      --skip {toxinidir}/.mypy_cache --skip {toxinidir}/icon.svg
```

Doesn't really change anything other than being less lines :sweat_smile:

Reply:
Well, technically codespell also skips hidden files and directories by default so those can be removed as well. However, to simplify this a bit more we should just move the skip directive to the `pyproject.toml` file, per [the docs](https://github.com/codespell-project/codespell#using-a-config-file):

```toml
[tool.codespell]
skip = "./build,./lib,./venv,./icon.svg"
```

This will, unfortunately, feel a bit incomplete to me as I suspect this list will always be approximately the contents of `.gitignore`. However, one thing at a time and its likely better to be explicit here in what we prefer to ignore rather than being implicit.

Reply:
Huh. I didn't know you could use pyproject.toml with codespell. I just added this to one of my own repos, but I *did* have to specify hidden directories, it seemed to keep checking `.git`, `.tox` etc unless I did:

https://github.com/jnsgruk/parca-k8s-operator/commit/627503e526e6b8c0eff7c9aa9cc4b6da48900c40

Reply:
> Huh. I didn't know you could use pyproject.toml with codespell. I just added this to one of my own repos, but I _did_ have to specify hidden directories, it seemed to keep checking `.git`, `.tox` etc unless I did:
> 
> [jnsgruk/parca-k8s-operator@627503e](https://github.com/jnsgruk/parca-k8s-operator/commit/627503e526e6b8c0eff7c9aa9cc4b6da48900c40)

@jnsgruk Are you using codespell with tox v4? Looking at your CI jobs for the parca-k8s-operator, it looks like you are. I noticed the same issue with codespell when I switched from tox v3 to tox v4.

Reply:
Latest commit should address this: [8899708](https://github.com/canonical/slurmd-operator/commit/8899708151557814e31f1aac8afdc1b91a8f992f)

Reply:
Looks good. Resolving this convo